web/src/components/SearchListFilters/SearchListFilters.test.tsx

```diff
@@ -186,9 +186,9 @@ describe("SearchListFilters", () => {
 			).not.toBeNull();
 		});
 
-		it("should render title filter input box and label", () => {
+		it("should render title filter input box and heading", () => {
 			expect(
-				screen.getByLabelText("Filter by title or keyword")
+				screen.getByPlaceholderText("E.g. 'diabetes' or 'NG28'")
 			).toBeInTheDocument();
 		});
```

Comment:
I think we can get rid of this test now actually, as it's the same as the test below it - now that we're not using a label that is.